.github/workflows/testing-new-gateway.yml

```diff
@@ -0,0 +1,107 @@
+name: Testing w/ new gateway depending on SDK changes
```

Comment:
I don't understand why we need this flow

Reply:
The gateway now depends on the sdk for stuff like DDO types and keeper, so to get the tests pass we might need to change both. (For example here Aitor changed the name of `nftType` to `ercType` in the DDO, and the gateway needs this param to determine which flow to use) To fix this we would have to have separate packages for stuff like data model and the keeper...

Reply:
probably for this kind of circular dependencies we could use something like this:
https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a

Steps:
1. we create a `sdk-js` branch with the breaking changes
2. we integrate that branch in a new gateway PR
3. at this point the gateway can be tested using the sdk breaking changes, the sdk can be tested using a gateway workflow in the CI
4. when both are green merge and re-adapt versioning

Reply:
That would work too, unfortunately including sdk-js doesn't work from a branch (that's why I need the tgz files), probably this could be fixed. I tried it once but there were always more and more problems so I gave up. :(

Reply:
Added here https://github.com/nevermined-io/sdk-js/issues/262